### PR TITLE
[web] Fix image gap due to svg element without position attribute

### PR DESCRIPTION
**Pull request by ferhatb**
## Description

- Fix position of svg filters in BitmapCanvas. This bug was encountered before in clipping svg paths but color filters caused the same issue in this case. Now using a constant header throughout for svg resources.

- dartfmt util.dart

## Related Issues

https://github.com/flutter/flutter/issues/59451

## Tests

I added the following tests:

color_filter_blendMode_overlay.png golden test.

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [contributor guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [C++, Objective-C, Java style guides] for the engine.
- [x] I read the [tree hygiene] wiki page, which explains my responsibilities.
- [x] I updated/added relevant documentation.
- [x] All existing and new tests are passing.
- [x] I am willing to follow-up on review comments in a timely manner.

## Reviewer Checklist

- [x] I have submitted any presubmit flakes in this PR using the [engine presubmit flakes form] before re-triggering the failure.

## Breaking Change

Did any tests fail when you ran them? Please read [handling breaking changes].

- [x] No, no existing tests failed, so this is *not* a breaking change.
- [ ] Yes, this is a breaking change. *If not, delete the remainder of this section.*
   - [ ] I wrote a design doc: https://flutter.dev/go/template *Replace this with a link to your design doc's short link*
   - [ ] I got input from the developer relations team, specifically from: *Replace with the names of who gave advice*
   - [ ] I wrote a migration guide: https://flutter.dev/go/breaking-changes-template *Replace this with a link to a pull request that adds the migration guide to https://flutter.dev/docs/release/breaking-changes*

### CODE REVIEW

**Review by mdebbar — APPROVED** [review]
lgtm

File: lib/web_ui/lib/src/engine/util.dart [hunk 1]
```diff
@@ -474,27 +494,27 @@ void applyWebkitClipFix(html.Element? containerElement) {
...
-      html.window.requestAnimationFrame((num _) {
-        _fontChangeScheduled = false;
-        window.invokeOnPlatformMessage(
-          'flutter/system',
-          _fontChangeMessage,
-              (_) {},
-        );
-      });
-    }
+  if (window._onPlatformMessage != null) if (!_fontChangeScheduled) {
```
Comment (nturgut) [inline lib/web_ui/lib/src/engine/util.dart:506]: very nit: curly braces around this if statement

File: lib/web_ui/test/golden_tests/engine/color_filter_golden_test.dart [hunk 2]
```diff
@@ -35,55 +36,118 @@ void testMain() async {
...
+        ..color = const Color.fromRGBO(0, 0, 255, 1)) as SurfacePaint);
+  return recorder.endRecording();
+}
+
+Picture _drawTestPictureWithImage(ColorFilter filter) {
+  final EnginePictureRecorder recorder =
+      PictureRecorder() as EnginePictureRecorder;
+  final RecordingCanvas canvas =
+      recorder.beginRecording(const Rect.fromLTRB(0, 0, 400, 400));
+  Image testImage = createTestImage();
```
Comment (nturgut) [inline lib/web_ui/test/golden_tests/engine/color_filter_golden_test.dart:112]: nit: final

**Review by nturgut — APPROVED** [review]

File: lib/web_ui/lib/src/engine/util.dart (hunk 1 above)
Comment (ferhatb) [reply lib/web_ui/lib/src/engine/util.dart]: Done.

File: lib/web_ui/test/golden_tests/engine/color_filter_golden_test.dart (hunk 2 above)
Comment (ferhatb) [reply lib/web_ui/test/golden_tests/engine/color_filter_golden_test.dart]: Done

**Pull request merged**